### ISSUE TITLE
Bug in classic when looting a boss

### ISSUE THREAD

**Issue by mcd8604** [evil-morfar/RCLootCouncil2#185]
This past week, we encountered a serious problem while looting Chrommagus. The master looter looted the boss corpse as he normally would have and the RC Council UI opened up for him as normal and initiated the player input queries from the raiders. However, one item was immediately (and automatically without notice) looted to a (random?) player in the raid. This player coincidentally and unfortunately happened to be one of our enchanters who proceeded to disenchant the item in a hurry without doubt. The item appeared to be still available for roll as it normally would be in the RC Council UI as if nothing had happened (to everyone including the master looter). We only noticed that it was looted automatically after a couple minutes when someone saw the loot entry in the chat log. I luckily grabbed a screenshot of the chat log: 
![image](https://user-images.githubusercontent.com/2568468/77810608-a88f5a00-706b-11ea-9a0c-5373dcfb7c2f.png)
https://media.discordapp.net/attachments/639326708578648084/692543409323507802/unknown.png?width=340&height=95

We are proceeding with multiple support tickets to get the item back to one of the appropriate players. I hope this is enough information to help debug. If not, please let me know what else I could provide to help out.

**Comment by evil-morfar** [comment]
If the item remained available to roll on then I wouldn't expect RCLootCouncil to have given it out. I will need to have a look at your SavedVariables though.

**Comment by mcd8604** [comment]
Thanks for responding so quickly. Our master looter and I took a look through the log data in the SavedVariables file. We noticed that the item that was looted out and then disenchanted is not in the log. I think this implies that RCLootCouncil is not at fault as it didn't process the item. We think that the master looter must have accidentally clicked on the default blizzard master looter UI without noticing and it assigned that item out.
